influxdb/2.3/Dockerfile

```diff
@@ -37,6 +37,7 @@ RUN set -eux && \
     gpg --batch --verify influxdb2-${INFLUXDB_VERSION}-linux-${ARCH}.tar.gz.asc influxdb2-${INFLUXDB_VERSION}-linux-${ARCH}.tar.gz && \
     tar xzf influxdb2-${INFLUXDB_VERSION}-linux-${ARCH}.tar.gz && \
     cp influxdb2_linux_${ARCH}/influxd /usr/local/bin/influxd && \
+    rm -rf influxdb2-* && \
```

Comment:
I think it should also remove `influxdb2_linux_${ARCH}` here. :upside_down_face:

Reply:
Thanks for the review. I'm braindead!